bundles/core/src/main/java/com/adobe/cq/forms/core/components/internal/models/v2/form/FormContainerImpl.java

```diff
@@ -64,6 +70,24 @@ public String getThankYouMessage() {
         return thankyouMessage;
     }
 
+    @Override
+    @Nullable
+    public String getClientLibRef() {
+        return clientLibRef;
+    }
+
+    @Override
+    @Nullable
```

Comment:
Author view?

Reply:
We would need this in author view for content tree use-case.